src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/CollectionSection.js

```diff
@@ -113,9 +113,10 @@ class CollectionSection extends React.Component<Props> {
     };
 
     handleSaveResponse = (resourceStore: ResourceStore) => {
+        const openedCollectionOperationOverlayType = this.openedCollectionOperationOverlayType;
         this.closeCollectionOperationOverlay();
 
-        if (this.openedCollectionOperationOverlayType === 'update') {
+        if (openedCollectionOperationOverlayType === 'update') {
```

Comment:
@alexander-schranz because of this bug the wrong resourceStore will be updated.

Reply:
@wachterjohannes Can we add a test for this, else somebody will revert it accidently in the future.